### ISSUE TITLE
[CI] Python PRs don't make use of python changes in CI runs

### ISSUE THREAD

**Issue by mqrause** [CleverRaven/Cataclysm-DDA#77479]
### Describe the bug

#77477 fixes an issue with a parser, but Text Changes Analyzer doesn't use the changes and fails with the old error anyway.

### Attach save file

n/a

### Steps to reproduce

n/a

### Expected behavior

Fixed issues are reflected in CI runs.

### Versions and configuration

n/a?

**Comment by Uwuewsky** [comment]
I believe CI will be fixed once the PR is merged with the master branch.

**Comment by mqrause** [comment]
Yes, but the point of this issue is that the PR didn't fix CI within the PR itself. By extension that also means the reverse case would be happening, so you could break CI with a python PR without noticing before it got merged.
